library/xwiki

```diff
@@ -3,17 +3,23 @@
 Maintainers: XWiki Core Dev Team <committers@xwiki.org> (@xwiki)
 GitRepo: https://github.com/xwiki-contrib/docker-xwiki.git
 
-Tags: 14, 14.2, 14.2.0, 14-mysql-tomcat, 14.2-mysql-tomcat, 14.2.0-mysql-tomcat, mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable, latest
+Tags: 14, 14.2, 14.2.1, 14-mysql-tomcat, 14.2-mysql-tomcat, 14.2.1-mysql-tomcat, mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable, latest
 Architectures: amd64
 Directory: 14/mysql-tomcat
-GitCommit: ce6faa3a58fc16e0eeca74038fbb43149fcf60b5
+GitCommit: 7e193b63d37ddf931c9bb3ab2401c805affa28d3
 
-Tags: 14-postgres-tomcat, 14.2-postgres-tomcat, 14.2.0-postgres-tomcat, postgres-tomcat, stable-postgres-tomcat, stable-postgres
+Tags: 14-postgres-tomcat, 14.2-postgres-tomcat, 14.2.1-postgres-tomcat, postgres-tomcat, stable-postgres-tomcat, stable-postgres
 Architectures: amd64, arm64v8
 Directory: 14/postgres-tomcat
-GitCommit: ce6faa3a58fc16e0eeca74038fbb43149fcf60b5
+GitCommit: 7e193b63d37ddf931c9bb3ab2401c805affa28d3
+
+Tags: 14-mariadb-tomcat, 14.2-mariadb-tomcat, 14.2.1-mariadb-tomcat, mariadb-tomcat, stable-mariadb-tomcat, stable-mariadb
+Architectures: amd64, arm64v8
+Directory: 14/mariadb-tomcat
+GitCommit: 7e193b63d37ddf931c9bb3ab2401c805affa28d3
 
-Tags: 14-mariadb-tomcat, 14.2-mariadb-tomcat, mariadb-tomcat, stable-mariadb-tomcat, stable-mariadb
+## Previous missing tag: this should be removed later.
```

Comment:
:heart_eyes: (thank you for noting what this was for; you successfully preempted my immediate question about it :smile: :+1:)